modules/virt-generalizing-linux-vm-image.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_custom/virt-creating-vms-uploading-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-generalizing-linux-vm-image_{context}"]
+= Generalizing a VM image
+
+You can generalize a {op-system-base-full} image to remove all system-specific configuration data before you use the image to create a golden image, a preconfigured snapshot of a virtual machine (VM). You can use a golden image to deploy new VMs.
+
+Generalizing a {op-system-base} VM is done using the `virtctl`, `guestfs`, and `virt-sysprep` tools.
+
+.Prerequisites
+
+* An {op-system-base} virtual machine.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. If the {op-system-base} virtual machine is running, stop it.
++
+[source,terminal]
+----
+$ virtctl stop <rhel-vm-name>
+----
+
+. Optional: Clone the virtual machine to avoid losing the data from your original VM. You can then generalize the cloned VM.
+
+. Retrieve the `dataVolume` that stores the root filesystem for the VM by running the following command:
++
+[source,terminal]
+----
+$ oc get vm <my-vm-name> -o jsonpath="{.spec.template.spec.volumes}{'\n'}"
+
+[{"dataVolume":{"name":"my-vm-volume"},"name":"rootdisk"},{"cloudInitNoCloud":{...}]
+----
+
+. Retrieve the Persistent Volume Claim (PVC) that matches the listed `dataVolume` by running the followimg command:
++
+[source,terminal]
+----
+$ oc get pvc
+NAME            STATUS   VOLUME  CAPACITY   ACCESS MODES  STORAGECLASS     AGE
+my-vm-volume Bound  …
+----
+
+[NOTE]
```

Comment:
<img width="703" alt="Screenshot 2024-10-08 at 3 02 48 PM" src="https://github.com/user-attachments/assets/287d5f36-7c3f-4c94-ad57-e02c9e2a1403">

This note is not rendering as a part of the Step.  I think using a + will make it aligned to the step.

For example like this - 

<img width="703" alt="Screenshot 2024-10-08 at 3 06 19 PM" src="https://github.com/user-attachments/assets/c406ea32-bf58-422f-8802-b188ea30236c">

Reply:
It's not meant to be part of the step, actually. It's not related to the step before it, but the steps after the note are necessary if the user is cloning a PVC instead of a VM. This was a tricky bit of documentation to make clear, and this was the best option. The devs were very happy with it.

Reply:
The note looks out of place. I don't think this pattern of rendering is allowed. However you can check on the channel if this note can be kept or need to be moved to another place...

Reply:
I'll indent it. I don't think it will matter for the end user.